app/models/applicant.rb

```diff
@@ -1,4 +1,14 @@
 class Applicant < ApplicationRecord
   has_many :applicant_pets
   has_many :pets, through: :applicant_pets
+  
```

Comment:
Using validation definitley makes more sense for the app instead of the required applicant strong params